x/ccv/consumer/keeper/relay.go

```diff
@@ -102,6 +115,8 @@ func (k Keeper) QueueVSCMaturedPackets(ctx sdk.Context) {
 
 		k.DeletePacketMaturityTimes(ctx, maturityTime.VscId, maturityTime.MaturityTime)
 
+		k.Logger(ctx).Info("VSCMaturedPacket enqueued", "vscID", vscPacket.ValsetUpdateId)
```

Comment:
```suggestion
		k.Logger(ctx).Debug("VSCMaturedPacket enqueued", "vscID", vscPacket.ValsetUpdateId)
```

(Packet enqueue is implementation detail, the info parts are when you send a packet IMO)

Reply:
Sending the packet is logged by the IBC module. Queueing the packet is important as that should result in an IBC send. Also, it's the only place where we can log the data without having to unmarshal it in `SendPackets`